spec.html

```diff
@@ -42955,6 +42966,383 @@ <h1>Properties of _TypedArray_ Instances</h1>
+                  1. Return the Record { [[Read]]: _read_, [[Bytes]]: _bytes_, [[Error]]: _error_ }.
+              1. Return the Record { [[Read]]: _length_, [[Bytes]]: _bytes_, [[Error]]: ~none~ }.
+            1. Let _char_ be the substring of _string_ from _index_ to _index_ + 1.
+            1. Set _index_ to _index_ + 1.
+            1. If _char_ is *"="*, then
+              1. If _chunkLength_ < 2, then
+                1. Let _error_ be a new *SyntaxError* exception.
+                1. Return the Record { [[Read]]: _read_, [[Bytes]]: _bytes_, [[Error]]: _error_ }.
+              1. Set _index_ to SkipAsciiWhitespace(_string_, _index_).
+              1. If _chunkLength_ = 2, then
+                1. If _index_ = _length_, then
+                  1. If _lastChunkHandling_ is *"stop-before-partial"*, then
+                    1. Return the Record { [[Read]]: _read_, [[Bytes]]: _bytes_, [[Error]]: ~none~ }.
+                  1. Let _error_ be a new *SyntaxError* exception.
+                  1. Return the Record { [[Read]]: _read_, [[Bytes]]: _bytes_, [[Error]]: _error_ }.
+                1. Set _char_ to the substring of _string_ from _index_ to _index_ + 1.
+                1. If _char_ is *"="*, then
+                  1. Set _index_ to SkipAsciiWhitespace(_string_, _index_ + 1).
+              1. If _index_ < _length_, then
+                1. Let _error_ be a new *SyntaxError* exception.
+                1. Return the Record { [[Read]]: _read_, [[Bytes]]: _bytes_, [[Error]]: _error_ }.
+              1. If _lastChunkHandling_ is *"strict"*, let _throwOnExtraBits_ be *true*.
+              1. Else, let _throwOnExtraBits_ be *false*.
+              1. Let _decodeResult_ be Completion(DecodeBase64Chunk(_chunk_, _throwOnExtraBits_)).
+              1. If _decodeResult_ is an abrupt completion, then
+                1. Let _error_ be _decodeResult_.[[Value]].
+                1. Return the Record { [[Read]]: _read_, [[Bytes]]: _bytes_, [[Error]]: _error_ }.
+              1. Set _bytes_ to the list-concatenation of _bytes_ and ! _decodeResult_.
+              1. Return the Record { [[Read]]: _length_, [[Bytes]]: _bytes_, [[Error]]: ~none~ }.
+            1. If _alphabet_ is *"base64url"*, then
+              1. If _char_ is either *"+"* or *"/"*, then
+                1. Let _error_ be a new *SyntaxError* exception.
+                1. Return the Record { [[Read]]: _read_, [[Bytes]]: _bytes_, [[Error]]: _error_ }.
+              1. Else if _char_ is *"-"*, then
+                1. Set _char_ to *"+"*.
+              1. Else if _char_ is *"_"*, then
+                1. Set _char_ to *"/"*.
+            1. If the sole code unit of _char_ is not an element of the standard base64 alphabet, then
+              1. Let _error_ be a new *SyntaxError* exception.
+              1. Return the Record { [[Read]]: _read_, [[Bytes]]: _bytes_, [[Error]]: _error_ }.
+            1. Let _remaining_ be _maxLength_ - the length of _bytes_.
```

Comment:
We say "the length of" for Strings, but "the number of elements in" for Lists. `bytes` is a List.

```suggestion
            1. Let _remaining_ be _maxLength_ - the number of elements in _bytes_.
```